pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -254,6 +256,33 @@ func (r *ReconcileNodeMaintenance) Reconcile(request reconcile.Request) (reconci
 	return reconcile.Result{}, nil
 }
 
+func makeBoolRef(val bool) (*bool) {
+	return &val;
```

Comment:
no need for () around the return value above, and the semicolon here

Reply:
done